icu4c/source/test/cintltst/unumberformattertst.c

```diff
@@ -212,6 +216,26 @@ static void TestExampleCode() {
 }
 
 
+static void TestSimpleNumberFormatter() {
```

Comment:
This is good, but please check that you call each new C function at least once from cintltst.

Reply:
Okay. I was calling every function, but not from the C test. I am now calling every function from the C test.

Reply:
Thanks!